### DISCUSSION TITLE
You don't have access to github Copilot.Join ...... [why?]

### DISCUSSION THREAD

**Discussion by tanzhenglian** [community/community#112683] [GitHub Education]
### Hi everyone,

I am on a student account and I also have to access to github copilot.
Is there any solution to this?

Thanks

**Comment by tanzhenglian** [comment]
<img width="436" alt="image" src="https://github.com/community/community/assets/6985625/3f4a75eb-9a2d-488e-8fe2-9bd5c7c832a6">

**Comment by msb-jr** [comment]
I have the same issue. Whenever I click "your copilot" in the dropdown on the right of github, it brings me to https://github.com/github-copilot/signup where it prompts me to pay for copilot, when I am a verified student (verified within two days ago).

I am wondering if it will update in some time, maybe it needs a bit from the time you're verified. 

When were you verified? Maybe the cause of our issue is the same?

**Reply by tanzhenglian** [reply]
02-26
